lib/config.js

```diff
@@ -152,6 +153,27 @@ const conf = convict({
       default: 10
     }
   },
+  openid: {
+    key: {
+      doc: 'Private JWK to sign id_tokens',
+      default: {}
+    },
+    oldKey: {
+      doc: 'The previous public key that was used to sign id_tokens',
+      default: {}
+    },
+    issuer: {
+      // this should match `issuer` in the 'OpenID Provider Metadata' document
+      // from the fxa-content-server
+      doc: 'The value of the `iss` property of the id_token',
+      default: 'https://accounts.firefox.com'
```

Comment:
I wonder if we should de-duplicate this somehow with the `contentUrl` config option.  Or, should the fact that the issuer is the content-server be considered an implementation detail?